src/lib/json2gettext.js

```diff
@@ -96,19 +97,17 @@ function parseGettext(domain, data, options = {}) {
       pArray.splice(getGettextPluralPosition(ext, kv.pluralNumber - 1), 0, kv.value);
 
       if (typeof trans[kv.context] !== 'object') trans[kv.context] = {};
+
       if (options.keyasareference) {
-        if (typeof trans[kv.context][kv.value] === 'object') {
-                      // same context and msgid. this could theorically be merged.
```

Comment:
It looks like this conditional isn't needed anymore? So I went ahead and removed it, which helped increase the test coverage - since the same `kv.key` is going to the same place in both branches of the conditional, and since the tests don't ever hit the positive case. But if the right solution should actually just be to add tests that *do* hit this case, definitely let me know, I'd imagine this might potentially be useful for someone? it's just not clear to me how exactly :woman_shrugging: